src/sentry/issues/grouptype.py

```diff
@@ -180,9 +180,22 @@ def __init_subclass__(cls: type[GroupType], **kwargs: Any) -> None:
         registry.add(cls)
 
         if not cls.released:
-            features.add(cls.build_visible_feature_name(), OrganizationFeature, True)
-            features.add(cls.build_ingest_feature_name(), OrganizationFeature)
-            features.add(cls.build_post_process_group_feature_name(), OrganizationFeature)
+            features.add(
```

Comment:
i think this is not doing what you want it to, this line would change the default to `True` as you've added an extra argument. could be helpful to used named parameters for these function calls.

additionally, do we need to modify any flagpole values to convert existing options over so that they remain the same when this merges?

Reply:
Ahh I see, looks like some of the other types manually registered the flag so I'll do the same.